src/sparseml/transformers/sparsification/trainer.py

```diff
@@ -831,39 +832,25 @@ def _generate_apply_manager_params(self, kwargs) -> Tuple[Optional[str], float]:
         return checkpoint, epoch
 
 
-class Trainer(TrainerInterface, TransformersTrainer):
+class TransformersTrainer(HFTransformersTrainer):
```

Comment:
Would be good to have an explicit init function here

Reply:
QQ: What benefit would that have?

Reply:
Currently we're not creating any instance from this class.